server/account/account.go

```diff
@@ -93,6 +97,9 @@ const (
 	// NoRule indicates that no rules have been broken. This may be an invalid
 	// value in some contexts.
 	NoRule Rule = iota
+	// PreimageReveal means an account failed to respond with a valid preimage
+	// for their order during epoch processing.
+	PreimageReveal
```

Comment:
I've been questioning if the `account` package is really the right place for `Rule` and conduct-related types.  But I prefer things not to move around without specific reason.  Was there an issue with the order?

Reply:
Oh, the ordering discussion was here, got it.  Still I think in a future change we're going to move `Rule` and related stuff into `auth` where it is all actually used.

Reply:
Yeah, I think just the order was different than the spec.